hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/schedule/ScheduleCreateSpecs.java

```diff
@@ -220,12 +226,14 @@ private HapiSpec bodyAndPayerCreation() {
                         .hasRecordedScheduledTxn());
     }
 
+    @HapiTest
     private HapiSpec bodyAndSignatoriesCreation() {
         var scheduleName = "onlyBodyAndSignatories";
         var scheduledTxn = cryptoTransfer(tinyBarsFromTo(SENDER, RECEIVER, 1));
 
         return defaultHapiSpec("BodyAndSignatoriesCreation")
                 .given(
+                        overriding(SCHEDULING_WHITELIST, "CryptoTransfer,CryptoCreate"),
```

Comment:
Was this override not necessary for mono code? Does it have to do with the comment further up about the weird required ordering of the tests?